### PR TITLE
Add Next and Recommended Articles to Article Page

### PR DESCRIPTION
**Pull request by nathanbegbie**
backend: https://praekeltorg.atlassian.net/browse/GEMMOLO-163
frontend: https://praekeltorg.atlassian.net/browse/GEMMOLO-199

### CODE REVIEW

**Comment by nathanbegbie** [comment]
@codiebeulaine please review

File: molo/core/models.py [hunk 1]
```diff
@@ -511,6 +511,22 @@ class SectionPage(CommentedPageMixin, TranslatablePageMixin, Page):
         null=True, blank=True,
         help_text='The date rotation will end')
 
+    enable_next_section = (
+        models.BooleanField(
+            default=False,
+            verbose_name='Activate up next section underneath articles',
```
Comment (codiebeulaine) [inline molo/core/models.py]: I think we can take out the up here

File: molo/core/models.py [hunk 2]
```diff
@@ -511,6 +511,22 @@ class SectionPage(CommentedPageMixin, TranslatablePageMixin, Page):
...
+            help_text=("Activate up next section underneath articles in this "
```
Comment (codiebeulaine) [inline molo/core/models.py]: here too

File: molo/core/models.py [hunk 3]
```diff
@@ -511,6 +511,22 @@ class SectionPage(CommentedPageMixin, TranslatablePageMixin, Page):
...
+                       "section will apear with the heading and subheading of"
```
Comment (codiebeulaine) [inline molo/core/models.py]: appear*

File: molo/core/models.py [hunk 4]
```diff
@@ -511,6 +511,22 @@ class SectionPage(CommentedPageMixin, TranslatablePageMixin, Page):
...
+                       " that article. The text will say 'next' in order to "
+                       "make the user feel like it's fresh content.")))
```
Comment (codiebeulaine) [inline molo/core/models.py]: Not sure if we need the explanation of what the text will say and why. Things like these are great to say to everyone when we are doing the demo

**Comment by nathanbegbie** [comment]
@codiebeulaine WRT the help text, it's copied over verbatim from the [Service Design SoW](https://media-api.atlassian.io/file/82911176-005c-4ba8-a4d4-b079116c4699/binary?token=eyJhbGciOiJIUzI1NiJ9.eyJpc3MiOiI3ZDBiZWMxNC1hOTNkLTQ2YzItYjg4Ny1mZTgwMzFiOTdjZWQiLCJhY2Nlc3MiOnsidXJuOmZpbGVzdG9yZTpmaWxlOjgyOTExMTc2LTAwNWMtNGJhOC1hNGQ0LWIwNzkxMTZjNDY5OSI6WyJyZWFkIl19LCJleHAiOjE0ODU3NTc1NDMsIm5iZiI6MTQ4NTc1Njg4M30.D4uwRdcDV4Slxyphk5OgMRTrJIQ3hOqOlosJyvdvm6o&client=7d0bec14-a93d-46c2-b887-fe8031b97ced&name=Related+and+Next+Articles+-+CMS+view.png) , so I don't think we should scrap it. I will update the text though because it changes it slightly. Also typos!

**Comment by codiebeulaine** [comment]
👍 wtih!

**Pull request merged**